### ISSUE TITLE
PEP8 check fails to consider ignores set in Spyder Settings

### ISSUE THREAD

**Issue by ma-sadeghi** [spyder-ide/spyder#14075]
### What steps will reproduce the problem?

PEP8 check fails to consider ignores set in Spyder Settings

## Versions

* Spyder version: 4.1.5 
* Python version: 3.8.3
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.5.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.16.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=1.0                   :  2.5.3 (OK)
pyls >=0.34.0;<1.0.0           :  0.34.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.5 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.1.2 (OK)
spyder_kernels >=1.9.4;<1.10.0 :  1.9.4 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.1 (OK)

# Optional:
cython >=0.21                  :  0.29.21 (OK)
matplotlib >=2.0.0             :  3.2.2 (OK)
numpy >=1.7                    :  1.18.5 (OK)
pandas >=0.13.1                :  1.0.5 (OK)
scipy >=0.17.0                 :  1.5.0 (OK)
sympy >=0.7.3                  :  1.6.1 (OK)
```

**Comment by ma-sadeghi** [comment]
![image](https://user-images.githubusercontent.com/14086031/97218311-e89ff200-179e-11eb-8aba-f6588af16a59.png)
![image](https://user-images.githubusercontent.com/14086031/97218369-fce3ef00-179e-11eb-98b9-d20ea5d1bf30.png)

**Comment by steff456** [comment]
Hi @ma-sadeghi,

We are currently working in fixing this issue and adding support to move the vertical line given the settings for the autocompletion engine. We expect to add this support in our next release.

Thanks for reporting!

**Comment by ccordoba12** [comment]
Hey @ma-sadeghi, a followup question for you: are you starting Spyder with an open project when this error happens?

**Comment by ma-sadeghi** [comment]
Hi @ccordoba12, I'm not 100% sure, but I almost always have an open project when I open Spyder, so I think yes.

**Comment by ccordoba12** [comment]
Ok, thanks for the confirmation. Then this is a duplicate of issue #13351 and it'll be fixed in Spyder 4.2.0, to be released in November.
